pkg/watchers/apiserver/apiserver.go

```diff
@@ -16,34 +16,38 @@ import (
 	fm "github.com/microsoft/retina/pkg/managers/filtermanager"
 	"github.com/microsoft/retina/pkg/pubsub"
 	"go.uber.org/zap"
+	"k8s.io/client-go/rest"
 	kcfg "sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
 const (
 	filterManagerRetries = 3
+	hostLookupRetries    = 3
```

Comment:
Do we need two different retry counter? I don't imagine needing such finer level of control.